Alloy/commands/compile/parsers/Alloy.Require.js

```diff
@@ -128,17 +128,21 @@ function parse(node, state, args) {
 			type === 'widget' ? 'Alloy.Widget' : 'Alloy.Require',
 			args.createArgs,
 			state
-		) + ');\n';
+		) + ')';
+	let parent = { symbol: args.symbol };
 	if (args.parent.symbol && !state.templateObject && !state.androidMenu) {
-		code += args.symbol + '.setParent(' + args.parent.symbol + ');\n';
+		code += ';\n' + args.symbol + '.setParent(' + args.parent.symbol + ');\n';
+		parent = {
+			symbol: args.symbol + '.getViewEx({recurse:true})'
+		};
+	} else {
+		code += '.getViewEx({recurse:true});\n';
 	}
 
 	return {
-		parent: {
-			symbol: args.symbol + '.getViewEx({recurse:true})'
-		},
+		parent: parent,
 		controller: args.symbol,
 		styles: state.styles,
 		code: code
-	};
+		};
```

Comment:
<!--
  1 failure:  Alloy/commands/co...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Alloy/commands/compile/parsers/Alloy.Require.js;
  Line: 147;
-->
- :no_entry_sign: Alloy/commands/compile/parsers/Alloy.Require.js line 147 – Expected indentation of 1 tab but found 2. (indent)